app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,11 +3,32 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  assignedContact: null,
+  tags: null,
```

Comment:
Couldn't we make the default value for tags as empty array []? Wouldn't that be more appropriate from a business perspective?

Currently let's say we add two tags. Then we remove both of them. Then the value will be an empty array and not a null right?

Reply:
Your suggestion is correct. Earlier when I made the default value for `tags` as empty array it showed an error. After trying out some more possibilities, when I added `min` constraint for the `tags` and the default value of `tags` as `null` it worked. But now as per your suggestion when I changed the value of `tags` to empty array keeping the `min` constraint it worked. Earlier I didn't had tried out this possibility.